string gripper_cmd
---
bool gripper_reply